docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md

```diff
@@ -0,0 +1,45 @@
+# Sites
+
+## Overview Study
```

Comment:
change to "Terminology"  ("Overview Study" is kind of inaccurate)

Reply:
its not terminology though, just `Overview` ?

Reply:
@ridz1208  agreed, just `Overview` would be fine